client/package.json

```diff
@@ -9,57 +9,55 @@
     "lint": "ng lint",
     "e2e": "ng e2e",
     "cy:open": "cypress open",
-    "docker-build": "npm run build -- --configuration production"
+    "docker-build": "npm run build"
```

Comment:
If you are going to change this you need to apply the changes to `angular.json` to make production the default config. See https://github.com/UMM-CSci-3601/3601-iteration-template/commit/9234ad74aac979a048fcb1c1376cc42e4cb61c98 and https://github.com/UMM-CSci-3601/3601-lab3_angular-javalin/pull/767

Reply:
Actually why do we have this at all? the Dockerfiles were removed, this line can be removed.

Reply:
Yeah, I was a little confused about why the Docker stuff was here. Presumably it got carried over from the iteration template at some point.

Reply:
Thanks for creating the issue #35 that lists everything that needs to be removed for this!